presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcMetadata.java

```diff
@@ -224,6 +224,11 @@ public JdbcTableHandle getTableHandle(ConnectorSession session, SchemaTableName
 
         JdbcTableHandle handle = (JdbcTableHandle) table;
 
+        if (!jdbcClient.supportsAggregationPushdown(session, handle, aggregates, assignments, groupingSets)) {
```

Comment:
You can put this after hard limits (after `if (handle.getGroupingSets().isPresent()) {` condition)